interface/main/main_title.php

```diff
@@ -87,6 +87,12 @@ function showhideMenu() {
             <a href='' class="css_button_small" style="margin:0px;vertical-align:top;display:none;" id='clear_active' onClick="javascript:parent.left_nav.clearactive();return false;">
```

Comment:
What is the above database.sql file doing. Should be modifying the file sql/database.sql instead.